benefits/core/migrations/0002_data.py

```diff
@@ -214,11 +203,7 @@ def load_data(app, *args, **kwargs):
         auth_provider=None,
         selection_label=_("eligibility.pages.index.mst_cc.label"),
         selection_label_description=_("eligibility.pages.index.mst_cc.description"),
-        start_title=_("eligibility.pages.start.mst_cc.title"),
-        start_headline=_("eligibility.pages.start.mst_cc.headline"),
-        start_item_heading=_("eligibility.pages.start.mst_cc.start_item.heading"),
-        start_item_details=_("eligibility.pages.start.mst_cc.start_item.details"),
-        start_help_anchor="mst-courtesy-card",
+        start_template="eligibility/start__mst_courtesycard.html",
```

Comment:
This is really nitpicky 😆 but in the app, on Python files, we hyphenate `courtesy_card`, like how `login_gov` is hyphenated.

Reply:
[cdfc32c](https://github.com/cal-itp/benefits/pull/1488/commits/cdfc32c77c3d75bb86393ca8fa583d344721da1f)